applications/CoSimulationApplication/python_scripts/solver_wrappers/kratos/kratos_base_wrapper.py

```diff
@@ -98,8 +102,21 @@ def _CreateAnalysisStage(self):
 
     def __GetAnalysisStage(self):
         if self.settings["solver_wrapper_settings"].Has("analysis_stage_module"):
-            analysis_stage_module = import_module(self.settings["solver_wrapper_settings"]["analysis_stage_module"].GetString())
-            return analysis_stage_module.Create(self.model, self.project_parameters)
+            module_name = self.settings["solver_wrapper_settings"]["analysis_stage_module"].GetString()
+            analysis_stage_module = import_module(module_name)
+            if hasattr(analysis_stage_module, "Create"):
+                return analysis_stage_module.Create(self.model, self.project_parameters)
+            else:
+                KM.Logger.PrintWarning("KratosBaseWrapper", "The analysis_stage_module (" + module_name + ") does not have a \"Create\" function, trying to create the AnalysisStage directly...")
+                if self.settings["solver_wrapper_settings"].Has("analysis_name"):
+                    analysis_stage_name = self.settings["solver_wrapper_settings"]["analysis_name"].GetString()
+                else:
+                    # We assume that the name of the AnalysisStage is the same as the name of the module in PascalCase instead of cammel_case
+                    file_name = module_name.split(".")[-1]
+                    # Convert Snake case to Pascal case
+                    analysis_stage_name = string.capwords(file_name.replace("_", " ")).replace(" ", "")
+                analysis = getattr(analysis_stage_module, analysis_stage_name)
```

Comment:
please add a proper error here if this does not exist

Reply:
Okay

Reply:
Done